locale/templates/auth.ftl

```diff
@@ -46,6 +47,10 @@ subplat-cancel-plaintext = { subplat-cancel }:
 subplat-reactivate = Reactivate subscription
 subplat-reactivate-plaintext = { subplat-reactivate }:
 subplat-update-billing = Update billing information
+subplat-privacy-policy = { -brand-mozilla } Privacy Policy
+subplat-privacy-policy-plaintext = { subplat-privacy-policy }:
```

Comment:
Do we need to expose this string to l10n?

Reply:
@xlisachan @LZoog

Reply:
These strings expose the same string as the non-plaintext version (and we want them to always be identical, which is why we're using a peaceable) but they also have a colon, so we do need them. This is the same for the other comments.

Reply:
@LZoog you are right, it contains  a`:` in addition to the brand name, so they should stay.